client/chatedit.cpp

```diff
@@ -70,6 +76,22 @@ void ChatEdit::contextMenuEvent(QContextMenuEvent *event)
     connect(action, &QAction::triggered, this, &KChatEdit::resetCurrentFormat);
     menu->addAction(action);
 
+    action = new QAction(QIcon::fromTheme("edit-paste"),
+                         tr(pastePlaintextByDefault() ? "Paste as rich text"
+                                                      : "Paste as plain text"),
```

Comment:
I don't think this works with Qt Linguist - it needs both strings to be there in separate, so that they land in separate translation slots.
```suggestion
                         pastePlaintextByDefault() ? tr("Paste as rich text")
                                                   : tr("Paste as plain text"),
```